src/app/projects/projects-list/projects-list.component.html

```diff
@@ -122,6 +124,11 @@
             <p *ngIf="!project.technologies.length">-</p>
           </ul>
         </td>
+
+        <td class="vf-table__cell">
+          {{project.cellCount}}<p *ngIf="!project.cellCount">-</p>
```

Comment:
Just out of curiosity, what happens if the cell count is 0:
{{project.cellCount}}<p *ngIf="!project.cellCount">-</p>

Reply:
What about using something like this:
```
<div *ngIf="showCellCount; else notProvided">{{project.cellCount}} </div>
<ng-template #notProvided>-</ng-template>
```

Reply:
do you not like this:
```
{{project.cellCount | number}}
<p *ngIf="project.cellCount === undefined || project.cellCount === null">-</p>
```

Reply:
You could also just do {{project.cellCount ?? '-'}} right? ?? syntax is like || but treats 0 as truthy